web/modules/custom/demo_content/content/rdf_entity/asset_distribution/blue.json

```diff
@@ -0,0 +1,189 @@
+{
```

Comment:
> I wouldn't put these fixtures in the modules themselves. Keeping everything centralized will assure an easy cleanup. (This module should be removed from the repo befor go live imho)

The module can be left there uninstalled. If a new developer comes, he can build/install the site and get some dummy content just for testing and developing. I don't see why this would be wrong.